### DISCUSSION TITLE
1.9.1版本的安装问题

### DISCUSSION THREAD

**Discussion by igoodful** [hhyo/Archery#2121] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/39941155/231246142-ae669a67-4ddb-4fd3-ba34-3048c35982e0.png)

**Comment by igoodful** [comment]
![图片](https://user-images.githubusercontent.com/39941155/231250156-3ccdd4e3-b959-42dc-a7c7-79445ce50e6a.png)

**Comment by hhyo** [comment]
提供一下部署和启动方式

同时参考 https://archerydms.com/FAQ/#_11

确认静态资源文件目录和权限是否正确

**Reply by igoodful** [reply]
感谢大佬的回复我的archery

**Reply by igoodful** [reply]
下面我给出详细的信息：
![image](https://user-images.githubusercontent.com/39941155/231463365-5ac73372-8dcf-4d00-ba5e-5969874abc74.png)
(venv) [root@a8-cloud-apple-db08 Archery]# cat archery/settings.py 
![image](https://user-images.githubusercontent.com/39941155/231464469-11710878-90ae-4865-afef-696ef576bab4.png)
![image](https://user-images.githubusercontent.com/39941155/231464550-ea1a8299-754e-4696-aadc-b7a0ce7efe5c.png)
![image](https://user-images.githubusercontent.com/39941155/231464627-81f2e94c-10ae-429d-a1fc-3d3c3f29f6d4.png)
![image](https://user-images.githubusercontent.com/39941155/231464703-80f55109-ccba-40a5-b438-8a5f0babd548.png)
![image](https://user-images.githubusercontent.com/39941155/231464768-69d0e40e-5561-41a5-b582-7e15c9290536.png)
![image](https://user-images.githubusercontent.com/39941155/231464830-1088dbdd-3bb8-4bc1-9e2b-0e354aec0406.png)
![image](https://user-images.githubusercontent.com/39941155/231464890-0d4a9eca-a6f8-475b-8df9-a6340bf66502.png)
![image](https://user-images.githubusercontent.com/39941155/231464951-e20bb071-e84e-4b34-8a5c-64f555a02d4c.png)

    
![image](https://user-images.githubusercontent.com/39941155/231463798-12ae90f6-998c-4743-a6cf-23b91780e12c.png)

**Reply by igoodful** [reply]
大佬，我安装了至少了10+次，虽然其中成功了一半，但这个问题我一直没有发现哪里问题，各自依赖包的安装，我看着都是安装成功的 ，mysql、redis等组件我都能确保安装成功且可用，这个问题我复现了很多次

**Reply by igoodful** [reply]
全程我都是以root身份执行的，按理说不会有权限问题

**Reply by hhyo** [reply, marked as answer]
可以看下ng的错误日志，应该是ng启动用户无static目录权限，可以做下授权，同时调整目录权限为755

**Reply by hhyo** [reply]
> 全程我都是以root身份执行的，按理说不会有权限问题

ng配置第一行就指定启动user了

**Reply by igoodful** [reply]
![image](https://user-images.githubusercontent.com/39941155/231479395-bf547085-960a-4a73-ba52-3b77de5acd19.png)

**Reply by igoodful** [reply]
万分感谢大佬的回答，解惑了我之前的安装出现错误的根本原因，这个错误太容易出现了，我很难想到这是nginx，尽管nginx配置文件第一行就是这个

![image](https://user-images.githubusercontent.com/39941155/231479974-db5f4ebf-aa4b-4c01-8679-46c03f21d4fc.png)
